starter-code/index.html

```diff
@@ -1,31 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  <meta charset="UTF-8">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+ <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" href="./css/main.css">
   <title>Instagram</title>
 </head>
+
 <body>
-  Instagram
+  <figure>
```

Comment:
figure is usually used to house elements that break away from the main flow of the document. Since this is an essential part of your page and very much within the main flow, you should use section instead.